test/test_torch.py

```diff
@@ -5634,6 +5634,64 @@ def test_index_add(self):
                             dest2[idx[i]] += src[i] * 2
                         self.assertEqual(dest, dest2)
 
+    # FIXME: move to indexing test suite
+    def test_index_reductions(self):
+        device = 'cpu'
+        reduces = ['mul', 'min', 'max', 'mean']
```

Comment:
You could use use parametrize as in [test_sparse_csr.py](https://github.com/pytorch/pytorch/blob/master/test/test_sparse_csr.py#L703-L708).